### DISCUSSION TITLE
What is an "asynchronous logger"?

### DISCUSSION THREAD

**Discussion by sketch34** [gabime/spdlog#2133] [Q&A] [answered]
I can take a guess but the documentation doesn't define it (that I could find so far). I want to know the details of what this means in the context of `spdlog`.

**Comment by tt4g** [comment, marked as answer]
Sink will process log messages in a dedicated thread.
Synchronous loggers block the calling thread until the I/O is finished when sink writes a log message to a file, but asynchronous loggers do not block.

See Wiki for more details: https://github.com/gabime/spdlog/wiki/6.-Asynchronous-logging
